src/Application/Redux/StateUtils.js

```diff
@@ -0,0 +1,157 @@
+import ImmutableAsset from './../Models/Asset';
+import { List } from 'immutable';
+import _ from 'underscore';
+
+const PRICE_INCREASE_PERCENTAGE = 7;
+const BASE_CLICK_INCOME = 1;
+
+const createNews = (news, newArticle) => {
+  const copyOfNews = news.slice();
+  copyOfNews.push(newArticle);
+  if (copyOfNews.length > 10) copyOfNews.shift();
```

Comment:
Understandable, a 3 line if block just looks ugly to me if there is only a single operation in there. If it is more readable as 3 lines I'll try to start doing that.